packages/target-nodejs-sdk/demo/proxy-test/proxy_demo.js

```diff
@@ -0,0 +1,107 @@
+const fetch = require("node-fetch");
+const HttpsProxyAgent = require("https-proxy-agent");
+// eslint-disable-next-line no-unused-vars
+const undici = require("undici");
+const TargetClient = require("../../dist/targetclient.server");
+
+let client;
+
+const context = {
+  channel: "web",
+  address: {
+    url: "http://local-target-test"
+  },
+  userAgent:
+    "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:73.0) Gecko/20100101 Firefox/73.0"
+};
+
+const visitorId = {
+  tntId: "338e3c1e51f7416a8e1ccba4f81acea0.28_0",
+  marketingCloudVisitorId: "07327024324407615852294135870030620007"
+};
+
+const targetRequest = {
+  id: visitorId,
+  context
+};
+
+function getOffers() {
+  client
+    .getOffers({
+      request: {
+        ...targetRequest,
+        prefetch: {
+          mboxes: [
+            {
+              name: "jason-flags",
+              index: 1
+            }
+          ]
+        }
+      },
+      sessionId: "dummy_session"
+    })
+    .then(res => {
+      console.log("Result: Success", JSON.stringify(res, null, 4));
+    })
+    .catch(err => {
+      console.log("ERROR");
+      console.log(err.message);
+    });
+}
+
+// proxy test for Node 18.2+ users
```

Comment:
I'd suggest having this one uncommented, and the ones for older Node versions commented out